corehq/apps/data_interfaces/tasks.py

```diff
@@ -74,7 +74,7 @@ def run_case_update_rules(now=None):
 
 @serial_task(
     '{domain}',
-    timeout=24 * 60 * 60,
+    timeout=36 * 60 * 60,
```

Comment:
it shouldn't, but in case it runs over i'm trying to make sure we won't spawn another one the next day